ansys/mapdl/core/errors.py

```diff
@@ -21,6 +21,10 @@
 """
 
 
+LICENSE_CONNECTION_ERROR = """
+ERROR CONNECTING TO LICENSE SERVER.
+"""
+
```

Comment:
```suggestion
```

This was moved into `LicenseServerConnectionError` since it's a short message.  Very long messages can be module globals, but one or two lines are ok to be local in the class or method.